copier/main.py

```diff
@@ -1232,7 +1232,7 @@ def _apply_update(self) -> None:  # noqa: C901
                         # The 3-way merge might have resolved conflicts automatically,
                         # so we need to check if the file contains conflict markers
                         # before storing the file name for marking it as unmerged after the loop.
-                        with Path(fname).open() as conflicts_candidate:
+                        with Path(fname).open(encoding="utf-8") as conflicts_candidate:
```

Comment:
Or how about opening the file with `"rb"` mode

```suggestion
                        with Path(fname).open("rb") as conflicts_candidate:
```

and turn the strings below into byte strings?

```diff
                            if any(
                                line.rstrip()
-                               in {"<<<<<<< before updating", ">>>>>>> after updating"}
+                               in {b"<<<<<<< before updating", b">>>>>>> after updating"}
                                for line in conflicts_candidate
                            ):
                                conflicted.append(fname)
```

This way, we'd be independent of encodings.